src/Symfony/Component/PasswordHasher/composer.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/password-hasher",
+    "type": "library",
+    "description": "Symfony Password Hashing Component",
```

```suggestion
    "description": "Provides password hashing utilities for both old and modern algorithms, along with hash migration capabilities",
```

(Symfony recently changed all its `description`s to be more descriptive)